.github/workflows/ruby.yml

```
@@ -5,6 +5,7 @@ on:
   workflow_dispatch:
 jobs:
   build:
+    if: ( github.event.schedule && github.repository == 'homebrew/homebrew-linux-fonts' ) || ! github.event.schedule
```

Can be simplified a bit:

```suggestion
    if: github.repository_owner == 'Homebrew' || !github.event.schedule
```